src/components/Navbar.tsx

```diff
@@ -78,6 +78,7 @@ const Navbar = ({
       `}
     />
     <ItemContainer>
+      <LinkItem href={urlMap.contact}>ABONNIEREN</LinkItem>
```

Comment:
do we want both ABONNIEREN and KONTAKT? or just one of the two?

Reply:
I think both, there is both functionality in the "footer".